src/Components/AddItem/AddItem.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react'
+import Select from 'react-select';
+import axios from 'axios';
+import { BsCheck2Circle } from 'react-icons/bs'
+import { AiOutlineClose } from 'react-icons/ai'
+import { FaExclamationCircle } from 'react-icons/fa'
+import './AddItem.css'
+
+function AddItem() {
+
+    const[inputs, setInputs] = useState({
+        product: '',
+        department: '',
+        packeging: ''
+    })
+
+    const departments = [
```

Comment:
Move all these consts outside the component

Reply:
do you mean still in the AddItem.js but outside of the function? if so, did it, but why actually do it?